### ISSUE TITLE
Make the conversation creation flow more lean

### ISSUE THREAD

**Issue by marcoambrosini** [nextcloud/spreed#10328]
**Is your feature request related to a problem? Please describe.**
The conversation creation dialog has become a very complex form that gets in the way of quickly getting a conversation started.

**Describe the solution you'd like**
Once the conversation is created, the content of the app is empty, so we can make use of the space for configuration purposes and make the dialog itself much leaner, or possibly even get rid of it. 
Instead, in the messages list there could be an empty-content with actions that allow me to configure this conversation for what exactly I need whether it's a quick meeting with a link, a big webinar, or a persistent group conversation with my team. 

**Additional context**
A great example of this is page creation in Notion the page is immediately generated, with no lag. And the empty-content looks great. Imagine how tedious it would be going through configurations and templates before creation. 

<img width="1004" alt="Screenshot 2023-08-24 at 08 10 12" src="https://github.com/nextcloud/spreed/assets/26852655/8a86a4c4-ac00-405e-825a-c640f4e4360d">

@nextcloud/designers thoughts?

**Comment by nimishavijay** [comment]
Sounds interesting, it also came up in one of the internal design feedback calls that there is a lot of info to decide when creating a conversation, so I would definitely be in favour of this 👍 The information shown directly could just be for eg. name, emoji, "Allow guests to join via link" + copy link button and the participants picker?

**Comment by DorraJaouad** [comment]
suggestion:  maybe a toggle button switching between  Private and Public  to replace

**Comment by marcoambrosini** [comment]
@nimishavijay I would like to try to experiment with requiring the least information possible before creation. Possibly just the name. 
If we put all the rest in the empty content we're not creating more friction for the user that wants to customize everything at the beginning, but we're removing substantial friction from the user that just wants to decide the bare minimum and maybe come back later for more tweaking.

**Comment by marcoambrosini** [comment]
As another reference, this is how an empty space (conversation) looks in Google chat

<img width="885" alt="Screenshot 2023-08-24 at 18 35 14" src="https://github.com/nextcloud/spreed/assets/26852655/ebde585a-5198-406c-afd7-5c4a1c907f22">

**Comment by jancborchardt** [comment]
Yeah definitely good point – both for MS Teams and Google Chat you can quickly just write something and then invite people, whereas we softly require people to add members first. Would be in favor of streamlining this too, good point @marcoambrosini :)
